src/django-app/django-project/quotes/serializers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+
+from .models import Quote, QuoteRequest, Status
+from shops.models import Shop
+
+class QuoteSerializer(serializers.ModelSerializer):
+    shop = serializers.PrimaryKeyRelatedField(queryset=Shop.objects.all())
+    quote_request = serializers.PrimaryKeyRelatedField(queryset=QuoteRequest.objects.all())
+    status = serializers.ChoiceField(choices=Status.choices)
```

Comment:
This would change to `status = serializers.ChoiceField(choices=Quote.Status.choices)`